2022-07-19 08:12:09,338 DEBUG    root            Loaded Command Group: ['gcloud', 'projects']
2022-07-19 08:12:09,345 DEBUG    root            Loaded Command Group: ['gcloud', 'projects', 'get_iam_policy']
2022-07-19 08:12:09,347 DEBUG    root            Running [gcloud.projects.get-iam-policy] with arguments: [PROJECT_ID_OR_NUMBER: "tradera-it-secrets"]
2022-07-19 08:12:21,593 DEBUG    google.auth.transport.requests Making request: POST https://oauth2.googleapis.com/token
2022-07-19 08:12:21,595 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): oauth2.googleapis.com:443
2022-07-19 08:12:21,841 DEBUG    urllib3.connectionpool https://oauth2.googleapis.com:443 "POST /token HTTP/1.1" 200 None
2022-07-19 08:12:21,878 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): cloudresourcemanager.googleapis.com:443
2022-07-19 08:12:22,303 DEBUG    urllib3.connectionpool https://cloudresourcemanager.googleapis.com:443 "POST /v1/projects/tradera-it-secrets:getIamPolicy?alt=json HTTP/1.1" 403 None
2022-07-19 08:12:22,307 DEBUG    root            (gcloud.projects.get-iam-policy) User [fredrik.larsson@tradera.com] does not have permission to access projects instance [tradera-it-secrets:getIamPolicy] (or it may not exist): The caller does not have permission
Traceback (most recent call last):
  File "/opt/google-cloud-sdk/lib/googlecloudsdk/calliope/cli.py", line 987, in Execute
    resources = calliope_command.Run(cli=self, args=args)
  File "/opt/google-cloud-sdk/lib/googlecloudsdk/calliope/backend.py", line 809, in Run
    resources = command_instance.Run(args)
  File "/opt/google-cloud-sdk/lib/surface/projects/get_iam_policy.py", line 48, in Run
    return projects_api.GetIamPolicy(project_ref)
  File "/opt/google-cloud-sdk/lib/googlecloudsdk/api_lib/cloudresourcemanager/projects_api.py", line 169, in GetIamPolicy
    return client.projects.GetIamPolicy(policy_request)
  File "/opt/google-cloud-sdk/lib/googlecloudsdk/third_party/apis/cloudresourcemanager/v1/cloudresourcemanager_v1_client.py", line 836, in GetIamPolicy
    return self._RunMethod(
  File "/opt/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 737, in _RunMethod
    return self.ProcessHttpResponse(method_config, http_response, request)
  File "/opt/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 743, in ProcessHttpResponse
    self.__ProcessHttpResponse(method_config, http_response, request))
  File "/opt/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 609, in __ProcessHttpResponse
    raise exceptions.HttpError.FromResponse(
apitools.base.py.exceptions.HttpForbiddenError: HttpError accessing <https://cloudresourcemanager.googleapis.com/v1/projects/tradera-it-secrets:getIamPolicy?alt=json>: response: <{'vary': 'Origin, X-Origin, Referer', 'content-type': 'application/json; charset=UTF-8', 'content-encoding': 'gzip', 'date': 'Tue, 19 Jul 2022 06:12:21 GMT', 'server': 'ESF', 'cache-control': 'private', 'x-xss-protection': '0', 'x-frame-options': 'SAMEORIGIN', 'x-content-type-options': 'nosniff', 'server-timing': 'gfet4t7; dur=246', 'alt-svc': 'h3=":443"; ma=2592000,h3-29=":443"; ma=2592000,h3-Q050=":443"; ma=2592000,h3-Q046=":443"; ma=2592000,h3-Q043=":443"; ma=2592000,quic=":443"; ma=2592000; v="46,43"', 'transfer-encoding': 'chunked', 'status': 403}>, content <{
  "error": {
    "code": 403,
    "message": "The caller does not have permission",
    "status": "PERMISSION_DENIED"
  }
}
>
2022-07-19 08:12:22,308 ERROR    root            (gcloud.projects.get-iam-policy) User [fredrik.larsson@tradera.com] does not have permission to access projects instance [tradera-it-secrets:getIamPolicy] (or it may not exist): The caller does not have permission
